packages/design-tokens/package.json

```diff
@@ -4,12 +4,48 @@
   "description": "Talend Design Tokens",
   "main": "lib/index.js",
   "types": "lib/index.d.ts",
-  "mainSrc": "src/index.ts",
+  "module": "./lib-esm/index.js",
+  "exports": {
+    "./package.json": "./package.json",
+    "./lib/tokens": {
+      "import": "./lib-esm/_tokens.scss",
```

Comment:
does it work to import a .scss file in native javascript esm, or is this consumed by a bundler?

Reply:
this is mainly to help webpack and the sass-loader to resolve tokens